### ISSUE TITLE
Allow partition devices in DEVS=

### ISSUE THREAD

**Issue by rhvgoyal** [projectatomic/docker-storage-setup#65]
Right now docker-storage-setup does not allow partition devices in DEVS=. For people who don't use lvm on root disk and don't create a volume group, this can be problematic. Now they don't have an extra disk and they can create a paritition on root disk and pass to dss.

If we allow partition devices in DEVS=, then one can easily create a partition on root disk and use that to carve out thin pool.

**Comment by bklau** [comment]
Some AWS instance are created from EBS and have no "root filesystem" LVM VG associated with it. So we  have: 
|<- boot partition->|<--root partition--.....>|
So we may need to partition the "root" partition into 2 partitions: one for the kernel use and one  to be used as block storage for devicemapper lvm-direct thinpool:
|<- boot partition->|<--root partition-->|<---free block partition for LVM VG--->|


Note: See an example here:
http://kumar-pravin.blogspot.com/2015/10/setting-up-storage-to-make-docker.html

**Comment by rhvgoyal** [comment]
So if you have a separate partition, you can simple create a volume group with that partition and pass that volume group to docker-storage-setup?

Say you have /dev/vdc1 partition free. Then do `vgcreate docker-vg /dev/vdc1` and pass `VG=docker-vg` in /etc/sysconfig/container-storage-setup.

**Comment by bklau** [comment]
Yes! The partition must be marked as "8e" partition type which is "Linux LVM"
